event/views.py

```diff
@@ -391,7 +379,7 @@ class EventUpdate(APIView):
         description="Update an event",
         responses={
             200: OpenApiResponse(
-                response=None, description="El evento se ha actualizado"
+                response=EventCreateSerializer(), description="El evento se ha actualizado"
```

Comment:
Esto debería de ser el EventSerializer no?

```suggestion
                response=EventSerializer(), description="El evento se ha actualizado"
```

Reply:
Hecho.